sdks/go/pkg/beam/pardo_test.go

```diff
@@ -0,0 +1,54 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package beam
+
+import (
+	"testing"
+)
+
+func TestRecommendParDo(t *testing.T) {
+	var tests = []struct {
+		name      string
+		outputDim int
+		want      string
+	}{
+		{"zero outputs", 0, "ParDo0"},
+		{"one output", 1, "ParDo"},
```

Comment:
Please add  3 more test cases here for 2 and 7,8 to cover those boundary conditions.

Reply:
Will do. Cheers.

Reply:
d9b5ef08e9